velox/functions/prestosql/tests/GreatestLeastTest.cpp

```diff
@@ -70,6 +74,15 @@ TEST_F(GreatestLeastTest, nanInput) {
       "Invalid argument to greatest(): NaN");
 }
 
+TEST_F(GreatestLeastTest, tryWithNan) {
+  runTest<double>(
+      "try(least(c0, 1.0))", {{0, 1.1, std::nan("1")}}, {0, 1.0, std::nullopt});
```

Comment:
Would you verify that without try there is an error? Use VELOX_ASSERT_THROW.

Reply:
There is a test above (nanInput) which does this - hence I did not add it again here.

Reply:
I see. Then, let's move this test next to the existing one.